packages/collapsible/src/Collapsible.tsx

```diff
@@ -1,248 +1,191 @@
+
+/* -------------------------------------------------------------------------------------------------
+ * CollapsibleTrigger
+ * -----------------------------------------------------------------------------------------------*/
+
+const TRIGGER_NAME = 'CollapsibleTrigger'
+
+type CollapsibleTriggerElement = TamaguiElement
+interface CollapsibleTriggerProps extends StackProps {
+  children: ReactNode | ((props: { open: boolean }) => ReactNode)
+}
+
+const CollapsibleTriggerFrame = styled(ThemeableStack, {
+  name: TRIGGER_NAME,
+  hoverTheme: true,
+  focusTheme: true,
+  pressTheme: true,
+  padded: true,
+  backgrounded: true,
+  bordered: true,
+  cursor: 'pointer',
+})
+
+const CollapsibleTrigger = React.forwardRef<
+  CollapsibleTriggerElement,
+  CollapsibleTriggerProps
+>((props: ScopedProps<CollapsibleTriggerProps>, forwardedRef) => {
+  const { __scopeCollapsible, children, ...triggerProps } = props
+  const context = useCollapsibleContext(TRIGGER_NAME, __scopeCollapsible)
+
+  return (
+    <CollapsibleTriggerFrame
+      tag="button"
+      aria-controls={context.contentId}
+      aria-expanded={context.open || false}
+      data-state={getState(context.open)}
+      data-disabled={context.disabled ? '' : undefined}
+      disabled={context.disabled}
+      {...triggerProps}
+      ref={forwardedRef}
+      onPress={composeEventHandlers(props.onPress as any, context.onOpenToggle)}
+    >
+      {typeof children === 'function' ? children({ open: context.open }) : children}
+    </CollapsibleTriggerFrame>
+  )
+})
+
+CollapsibleTrigger.displayName = TRIGGER_NAME
+
+/* -------------------------------------------------------------------------------------------------
+ * CollapsibleContent
+ * -----------------------------------------------------------------------------------------------*/
+
+// TODO: there might be something wrong with these two types
+type CollapsibleContentElement = TamaguiElement
+interface CollapsibleContentProps extends AnimatePresenceProps, ThemeableStackProps {
+  /**
+   * Used to force mounting when more control is needed. Useful when
+   * controlling animation with React animation libraries.
+   */
+  forceMount?: true
+}
+
+const CONTENT_NAME = 'CollapsibleContent'
+
+const CollapsibleContentFrame = styled(ThemeableStack, {
+  justifyContent: 'flex-start',
+  alignItems: 'center',
+  flexDirection: 'column',
+  name: CONTENT_NAME,
+  padded: true,
+  focusable: true,
+})
+
+const CollapsibleContent = React.forwardRef<
+  CollapsibleContentElement,
+  CollapsibleContentProps
+>((props: ScopedProps<CollapsibleContentProps>, forwardedRef) => {
+  const { forceMount, children, __scopeCollapsible, ...contentProps } = props
+  const context = useCollapsibleContext(CONTENT_NAME, __scopeCollapsible)
+
+  return (
+    <AnimatePresence {...contentProps}>
+      {forceMount || context.open ? (
+        <CollapsibleContentFrame {...contentProps}>{children}</CollapsibleContentFrame>
```

Comment:
It looks like the ID for linking the collapsible region to the trigger via `aria-controls` might be missing here?
```suggestion
        <CollapsibleContentFrame
          id={context.contentId}
          {...contentProps}
        >
          {children}
        </CollapsibleContentFrame>
```

Reply:
nice catch

Reply:
actually `aria-control` comes from props.